examples/clipping_planes_interactive_.py

```diff
@@ -120,7 +120,7 @@ def shift_plane_along_normal(viewer, event):
 
     # Get transform which maps from data (vispy) to canvas
     # note that we're using a private attribute here, which may not be present in future napari versions
-    visual2canvas = viewer.window._qt_viewer.layer_to_visual[volume_layer].node.get_transform(
+    visual2canvas = viewer.window._qt_viewer.canvas._layer_to_visual[volume_layer].node.get_transform(
         map_from="visual", map_to="canvas"
     )
```

Comment:
This shows the change in QtViewer class that is part of the public API. there should be backward compatibility (as it will be two lines)

Reply:
I could add backward compatibility by having layer_to_visual attribute in QtViewer referencing to canvas.layer_to_visual. First making canvas._layer_to_visual public. Does this sound ok?

Reply:
If so should the same be done for overlay_to_visual?

Reply:
Do you mean to add a property, not an attribute?

Reply:
ah yeah sorry, just a getter

Reply:
> If so should the same be done for overlay_to_visual?

No need, this is new stuff that was never released, so there is no backward complatibility to preserve.

Reply:
Resolved with [3fab498](https://github.com/napari/napari/pull/5432/commits/3fab498fa2525a31e1d914132a01683050683a79)